interfaces/quizoot.ts

```diff
@@ -262,8 +264,9 @@ export declare namespace Quizoot {
         surname?: string;
         /**
          * A pseudo for the quiz author.
+         * @nullable
          */
-        pseudo?: string | null;
+        pseudo?: string;
```

Comment:
this one is ok because `pseudo` is not required. we can even remove `@nullable`

Reply:
of course